### ISSUE TITLE
Ability to see latest change on a document

### ISSUE THREAD

**Issue by kevmodrome** [Budibase/budibase#2330]
**Describe the feature request**
I'm currently in the process of building an activity list of sorts and I find myself wanting to see the latest changes. Imagine a list like so:
- `Expense X was approved by Kevin`
- `Expense Y was denied by Joe`
- `Expense Z's amount was changed to $300 by Kevin`

**Comment by joebudi** [comment]
Can you not sort by the updated-at column (auto column)?

**Comment by kevmodrome** [comment]
Yeah, I can see which document has been changed but I want to see what was actually changed :)

**Comment by kevmodrome** [comment]
Could be something like a `Last change` column or something 🤔

**Comment by aptkingston** [comment]
Is this pretty much an audit log or do you think there's an easier middleground? Is there value in just storing the latest change?

**Comment by kevmodrome** [comment]
I think just having a "Last change" and using the "Updated at" in combination is good enough, at least for the use-case I had in mind; a recent activities list, pretty much. I figure I could probably work around it by creating a "History" table that gets pushed to every time a row gets changed or something 🤔

**Comment by shogunpurple** [comment]
https://pouchdb.com/guides/changes.html

This API is something I’ve been thinking we could use for a while. I know what you mean - it would be really cool to see like a bindable “recent actions”, but with the shape of the response here it could be hard to work with without client side JS in the builder
